spec/operators/audit-spec.ts

```diff
@@ -139,6 +139,18 @@ describe('Observable.prototype.audit', () => {
     expectSubscriptions(e1.subscriptions).toBe(e1subs);
   });
 
+  it('should mirror source if durations are Observable.of()', () => {
```

Comment:
I'd rather name it for synchronous observable instead of specifically say `of` operator (this could happen for any synchronous observable).